src/Features/Core/Portable/DocumentationComments/AbstractDocumentationCommentFormattingService.cs

```diff
@@ -289,6 +348,33 @@ public string Format(string rawXmlText, Compilation compilation = null)
             return state.GetText();
         }
 
+        public IEnumerable<TaggedText> Format(ISymbol symbol, SemanticModel semanticModel, int position, CancellationToken cancellationToken, SymbolDisplayFormat format = null)
+        {
+            var docSymbol = symbol;
+            if (symbol.GetTypeParameters().Any())
+            {
+                docSymbol = symbol.OriginalDefinition;
+            }
+            var rawXmlText = GetDocumentation(docSymbol, semanticModel.Compilation, cancellationToken);
+
+            if (rawXmlText == null)
+            {
+                return null;
```

Comment:
consider returning an empty enumerable here.

Reply:
this is how the other format methods defended against rawXmlText being null

Reply:
the other format method can go away :)   it's not used anywhere after your change in this PR.

Reply:
done